docs/manual/programmability/eval-intro.md

```diff
@@ -438,3 +438,13 @@ it still has a different set of defaults compared to a script without a `#!` lin
 Another difference is that scripts without `#!` can run commands that access keys belonging to different cluster hash slots, but ones with `#!` inherit the default flags, so they cannot.
 
 Please refer to [Script flags](/topics/lua-api#script_flags) to learn about the various scripts and the defaults.
+
+## Read-only scripts
+A read-only script is a script that only executes commands that don't modify any keys within Redis.
+Read-only scripts are useful because they can always be executed on replicas and can always be killed by the `SCRIPT KILL` command. 
+Read-only scripts can be executed either by adding the `no-writes` flag to the script or by executing the script with one of the read-only script command variants: `EVAL_RO`, `EVALSHA_RO`, or `FCALL_RO`.
+In addition to the benefits provided by all read-only scripts, the read-only script commands are also not blocked during write pauses, such as those that occur during coordinated failovers, and can be used to configure an ACL user to only be able to execute read-only scripts.
```

Comment:
this will no longer be true after we merge https://github.com/redis/redis/pull/10744
let's drop it the first part of that sentence (the part that says only the RO commands have that benefit)

Reply:
i'll merge this as is, and handle it in #1954